### ISSUE TITLE
Hidden videos like on Youtube without logging in.

### ISSUE THREAD

**Issue by skaldenhoven** [WWBN/AVideo#5753]
I couldn't find anything about this specific functionality in the open issues or the wiki so first as a question and if needed I can change this in a feature request.

I can create Private videos by deactivating Public Media and adding groups who can watch this video. However these persons need to be logged into AVideo which is something I don't want.
I would really prefer to have it hidden on the site for users not logged in and only be available for users with the direct link or on pages where I have embedded this. (Those pages are already password protected)
I know I can password protect a video and it will be visible but won't be accessible for users not logged in. However when embedding it on a page with password protection I do not want it to present another password prompt.

So in short would it be possible to hide videos on the main page for users not logged in while still being able to access the video with a direct link.

**Comment by DanielnetoDotCom** [comment]
Did you try this?
![chrome-capture (2)](https://user-images.githubusercontent.com/18009293/135110076-10b34ffd-46af-4ee4-b58d-c67d40698f09.gif)

**Comment by skaldenhoven** [comment]
Yes. Sorry. My Bad. So focused on Password / Groups that I forgot this issue.
